### DISCUSSION TITLE
add syntax to Dockerfiles

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#4211] [Ideas] [answered]
see https://docs.docker.com/engine/reference/builder/#syntax

**Comment by szaimen** [comment]
Hi, thanks for the suggestion. Sounds interesting! Would you mind opening a PR? :)

**Reply by Zoey2936** [reply]
do you want to use `# syntax=docker/dockerfile:1` or `# syntax=docker/dockerfile:latest` or `# syntax=docker/dockerfile:labs` or `# syntax=docker/dockerfile:1-labs`, etc?

**Reply by szaimen** [reply]
`# syntax=docker/dockerfile:latest`

I suppose this one...

`# syntax=docker/dockerfile:1`

Or this if we can automatically update it if a new version comes out...

**Reply by Zoey2936** [reply, marked as answer]
see #4232

**Reply by szaimen** [reply]
thx :)
